sycl/source/detail/scheduler/commands.cpp

```diff
@@ -3620,6 +3620,34 @@ ur_result_t ExecCGCommand::enqueueImpQueue() {
             MQueue->getHandleRef(), SemSignal->getExternalSemaphore(),
             OptSignalValue.has_value(), SignalValue, 0, nullptr, nullptr);
   }
+  case CGType::AsyncAlloc: {
+    // NO-OP.
```

Comment:
This comment is too short to be helpful, it should say why it's a no-op. i.e, why are we calling `urEnqueueUSMFreeExp` for the free CGType, but not UR alloc for this CG.